databook/background/Mouse-visual-system.md

```diff
@@ -45,6 +45,26 @@ visual areas each contain a distinct map of {term}`Retinotopy`, i.e. each visual
 area has its own map of visual space. The borders between higher visual areas
 can be identified based on reversals in the retinotopic map at area boundaries.
 
+The retinotopic map is often studied using intrinsic signal imaging (ISI), which
+utilizes the difference in the absorption spectra between deoxygenated and
+oxygenated hemoglobin. The cortical surface is exposed to red light and the
```

Comment:
Let's delete the two sentences starting at "The cortical surface is exposed... " to the end of the paragraph and merge with the next paragraph.